expenses-manager-frontend/src/main/frontend/src/app/daily-stats/daily-stats.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="dashboard-container">
+
+  <div class="date-picker-container">
```

Comment:
As integra date picker-ul in chart-container, asa cum este in figma cu o linie sub date picker